src/card/card.js

```diff
@@ -0,0 +1,73 @@
+import {LitElement, html} from 'lit-element/lit-element';
+
+/**
+ * display a single concept (heading, description, cta, icon) in a small UI element
+ */
+export default class BULCard extends LitElement {
+
+  // don't need 'slot' functionality, so lets use Light DOM
+  createRenderRoot(){ return this; }
+
+  static get properties() {
+    return {
+      /** name and primary interactable link in the card */
+      title: {type: String},
+      /** short bit of text describing the main action/concept */
+      description: {type: String},
+      /** material icon displayed to the left of the  */
+      icon: {type: String},
+      /** optional link for the href/window.open() action */
+      link: {type: String},
+      /** custom javascript */
+      js: {type: String},
+      
+      debug: {type: Boolean}
+    };
+  }
+
+  render() {
+    let href = !!this.js ? "javascript:void(0);" : this.link;
+    let action = !!this.js ? this.js : `window.open('${this.link}', '_self')`;
+    return html`
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"> <!-- @todo: replace with npm install -->
+      <style>
+        /* cards */
+        .deck, .ctas {
+          display: flex;
+          flex-wrap: wrap;
```

Comment:
should use the published css version here instead of doubling up. (eventually we'll want to grab the css directly, but we'll see)